EventFilter/FEDInterface/interface/FED1024.h

```diff
@@ -112,27 +112,36 @@ namespace evf{
 
     class TCDSBST{
     public:
-      struct bst{
-	uint32_t gpstimelow;
-	uint32_t gpstimehigh;
-	uint32_t low0;
-	uint32_t high0;
-	uint32_t low1;
-	uint32_t high1;
-	uint32_t low2;
-	uint32_t high2;
-	uint32_t low3;
-	uint32_t high3;
-	uint32_t low4;
-	uint32_t high4;
-	uint32_t low5;
-	uint32_t status;
+      union tcdsbst {
+        uint64_t words[8];
+        struct {
+          uint32_t gpstimelow;
```

Comment:
@slava77 replying to this now :-)
I had to fight against it when I re-implemented the `TCDSBST` class to look like the others in this header file. The `bst` POD-like struct does not suffer any alignment issues thanks to the way I have put the data members inside. As you can see, some data members are not "trivially" cast to a real physics value. AFAICT padding came in as soon as I was using `uint8_t` data members, or for "odd" combinations of `uint32_t` and `uint16_t` (as expected). Instances of this class are explicitly cast to a memory area when [constructing a `TCDSRecord` instance](https://github.com/diguida/cmssw/blob/3e85ca3ca923b8ce1fc3a5868232f4c34e9fe09e/EventFilter/FEDInterface/interface/FED1024.h#L149-L154) using `sizeof` function.
I let the DAQ experts comment further.